### DISCUSSION TITLE
Project Beta: Allow issue grouping by label

### DISCUSSION THREAD

**Discussion by alindst** [community/community#9346] [Projects and Issues]
Allow issues to be grouped by label value just like most of the other field/columns. Lacking that, I may have to repurpose milestone for the labels I want to group on which is inconsistent with how I use milestones across repos.

**Comment by iamhenry** [comment]
+1

has there been an update on this?

**Reply by khismatullin** [reply]
+1

**Comment by tay1orjones** [comment]
This looks like a duplicate of https://github.com/orgs/community/discussions/9172
